bots/pr/src/main/java/org/openjdk/skara/bots/pr/LabelCommand.java

```diff
@@ -36,7 +36,7 @@ public class LabelCommand implements CommandHandler {
 
     private static final Pattern argumentPattern = Pattern.compile("(?:(add|remove)\\s+)((?:[A-Za-z0-9_@.-]+[\\s,]*)+)");
     private static final Pattern shortArgumentPattern = Pattern.compile("((?:[-+]?[A-Za-z0-9_@.-]+[\\s,]*)+)");
-    private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk.java.net)?)$");
+    private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk.org)?)$");
```

Comment:
The literal dot in `openjdk.org` inside the `Pattern` should probably be escaped
`private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk\\.org)?)$");`
I didn't check if there are other occurrences, though.

Reply:
While you are technically correct, this was not done before either. I don't think we should combine a massive, but trivial, search-and-replace PR with additional hands-on-fixes like that.